my-wireframe/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wire frame</title>
+    <link rel="stylesheet" href="css/index.css">
+</head>
+
+<body>
+    <header>
+        <h1>Git Essentials: A Guide to Version Control for Developers</h1>
+        <p><b>Git Essentials</b> is a beginner-friendly site  designed to explain the fundamental concepts of
```

Comment:
```suggestion
        <p><cite>Git Essentials</cite> is a beginner-friendly site  designed to explain the fundamental concepts of
```

Reply:
Thank you for the suggestion. I have replace the b with cite tag.